constants.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
This file should have a license header block just like other shell scripts, even though it only has constants (for now).

Reply:
My bad, I added this file only recently so forgot completely. Good catch 👍